pyproject.toml

```diff
@@ -36,6 +36,8 @@ python = "^3.6"
 celery = "~5.1.0"
 # Fundamental web framework for Nautobot
 Django = "~3.1.12"
+# Adds AJAX capabilities to django-tables2
+django-ajax-tables = "~1.1.1"
```

Comment:
As discussed during daily stand-up, if we are agreeable to including this library, I think that we should just cargo cult it and embed it in the core since it is MIT-licensed and hasn't been updated in nearly 3 years. 

If there are no objections to that, after this PR is approved, I will include license attribution in the `NOTICE` file and make the necessary changes. I just didn't want to waste the effort to do that yet if it wasn't the path forward.